users.py

```diff
@@ -73,8 +73,49 @@ def _delete(self, user_id):
         found_user.delete()
 
 
+class SettingsHandler(common.ContentHandler):
+
+  def load_user_pref(self):
```

Comment:
This seems like a pretty common case.  Would it make sense to create some type of base handler class that includes this logic that folks like SettingsHandler could depend on?

Reply:
I moved it into models.UserPref so that it can be called from any handler that needs it.
I don't like to build up too much logic in base classes when there is another good place to put it.